src/api/app/controllers/webui/package_controller.rb

```diff
@@ -10,6 +10,7 @@ class Webui::PackageController < Webui::WebuiController
 
   before_action :set_project, only: [:show, :index, :users, :linking_packages, :dependency, :binary, :binaries,
                                      :requests, :statistics, :commit, :revisions, :submit_request_dialog,
+                                     :prefill,
```

Comment:
:thought_balloon: I really don't like this naming. Should include some reference to what is actually prefilled. It's not the package...

Reply:
Hm, good point.

Reply:
How about `:prefill_submit_request_form`?

Reply:
Hm, maybe something more generic like `:branch_diff_info`?

Reply:
I like it